elastic/elasticsearch
# Implement Enterprise Search GET Engine API request

Adds a GET Engine API call to the Enterprise Search Engines module. 

To test this: 
1. Enable the feature flag `xpack.ent-search.enabled=true` and ensure all required licensing and authentication is set. 
2. Create an Engine. Using the PUT API (outside this PR) execute a call to upsert the engine. This will automatically create the required system index and alias. Example: 
```
curl --location --request PUT 'http://localhost:9200/_engine/puggles' \
--<credentials>
--header 'Content-Type: application/json' \
--data-raw '{
    "indices": ["test1", "test2"]
}'
```

3. Get the Engine. Example: 
```
curl --location --request GET 'http://localhost:9200/_engine/puggles' \
--<credentials>
--header 'Content-Type: application/json'
```
